static/install.html

```diff
@@ -106,6 +107,10 @@ <h3 id="obtaining-fastboot">
                     tools, although they aren't necessary to install) and
                     <code>android-udev</code> for udev rules to make fastboot / adb work in local
                     sessions without root</li>
+                <li>Manjaro Linux: is a friendly and open-source Linux distribution based
```

Comment:
Can you avoid describing the distribution here? I don't want to advertise it.

Reply:
Note taken, thank you for the feedback.